src/Task/PhpMd.php

```diff
@@ -64,7 +66,8 @@ public function run(ContextInterface $context): TaskResultInterface
 
         $arguments = $this->processBuilder->createArgumentsForCommand('phpmd');
         $arguments->addCommaSeparatedFiles($files);
-        $arguments->add('text');
+        $format = in_array($config['report_format'], ['text', 'ansi']) ? $config['report_format'] : 'text';
```

Comment:
It's better to do these kind of normalizations inside the options resolver directly:
https://symfony.com/doc/current/components/options_resolver.html#option-normalization